.gitignore

```diff
@@ -1,2 +1,7 @@
 target/
-Cargo.lock
```

Comment:
Do we want `Cargo.lock` in this repo? The convention is to not include it for libraries I think.

Reply:
Alrighty. I'll remove it.